front-end/src/common/utils.ts

```diff
@@ -33,4 +33,16 @@ export function SaveBlob(blob: any, fileName: string) {
 
   // Clean up and remove the link
   link.parentNode?.removeChild(link);
+}
+
+export function nth(n: number): string {
+  return["st","nd","rd"][((n+90)%100-10)%10-1]||"th"
+}
+
+export function getNumberWithSuffix(n: number): string {
+  return `${n}${nth(n)}`
+}
+
+export function getMinutesFromRiotTime(riotTime: number): number {
```

Comment:
I hate to do this to you. But https://github.com/afieldi/RisenStats/blob/master/Common/utils.ts this already exists.

Reply:
PAIN